src/module-elasticsuite-core/etc/elasticsuite_analysis.xml

```diff
@@ -165,6 +177,7 @@
                 <filter ref="word_delimiter" />
                 <filter ref="lowercase" />
                 <filter ref="elision" />
+                <filter ref="standard_override" />
```

Comment:
Change name to "standard_before" ? Or something that does not directly reference the fact that it will be a stemmer_override (as "standard" does not say that the token filter will be a stemmer) ?